### PR TITLE
Cisme

### PR DESCRIPTION
**Pull request by wickwitch**
Hello Khanh,

I have just done:
- The tests for models Likes and Comments.
- Update the version of rspec-rails to 7.0

Please do some review and merge if it is OK.
Thanks

### CODE REVIEW

File: spec/models/topic_spec.rb
Comment (khanh-devos) [inline spec/models/topic_spec.rb:1]: Thanks for your modifying the testing of model user.

**Review by khanh-devos — CHANGES_REQUESTED** [review]
Well done Cisme :+1: 
Please replace some single quotes '' with double quite "" and add some one space inside the curly brackets like this pattern { text }.

File: spec/models/topic_spec.rb
Comment (wickwitch) [reply spec/models/topic_spec.rb:1]: You're welcome. We are one team.

**Comment by wickwitch** [comment]
> Well done Cisme 👍 Please replace some single quotes '' with double quite "" and add some one space inside the curly brackets like this pattern { text }.

Dear Khanh,
I could see the curly brackets inside the spec folder.
But could you tell me which file containing single quotes? I did not find them.

Thanks Khanh

**Comment by khanh-devos** [comment]
> > Well done Cisme 👍 Please replace some single quotes '' with double quite "" and add some one space inside the curly brackets like this pattern { text }.
> 
> Dear Khanh, I could see the curly brackets inside the spec folder. But could you tell me which file containing single quotes? I did not find them.
> 
> Thanks Khanh

No problem Cisme, 
The single quotes are located inside Gemfile.
By the way, just see them here at the tab "Files changed".

**Review by khanh-devos — APPROVED** [review]
Bravo Cisme :100: 
You can merge now.

**Pull request merged**